### PR TITLE
Trails project: Code review

### PR DESCRIPTION
**Pull request by helio3197**
Added the trails project's files for review.

### CODE REVIEW

**Review by helio3197 — COMMENTED** [review]
I think you would need to add the linters files and replace the divs tags with the semantic ones like &lt;header&gt;, and add a proper README documentation.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
-# trials-project-review
-Code review project
+# FCC Test Suite Template
```
Comment (helio3197) [inline README.md:1]: The README 's sections like "Built With" and "Getting Started" are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  <body>
+      <div id="main">
+      <h1 id="title">Nelson Mandela's tribute page</h1>
+      <p>This is collaborative project</p>
+      <figure id="img-div">
+        <img id="image" src="https://www.un.org/en/events/mandeladay/assets/img/featured-image-tribute.jpg" alt="Nelson Mandela's tribute" width="100%">
+        <figcaption id="img-caption">
+          Nelson Mandela showed what is possible for our world and within each one of us — if we believe, dream and work together. Let us continue each day to be inspired by his lifelong example and his call to never cease working for a better and more just world. 
+        </figcaption>
+      </figure>
```
Comment (helio3197) [inline index.html:18]: Why don't you wrap all these lines of code inside of an &lt;header&gt; that would improve the HTML and semantic structure and make the page more accessible.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @helio3197, 

Good job so far!

## Highlight!
- Good use of inline comment and review changes features ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (julie-ify) [reply README.md:1]: Good job with the review!. However, we want our comments to be clear and easy to understand. For Example:

This is a clear comment: The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?

This is not a clear comment: You would like to change the documentation and add a formal and professional description.

File: index.html (hunk 2 above)
Comment (julie-ify) [reply index.html:18]: Nice one here!! However, Let's make sure that we always include an explanation, a "why", for each requested change. For example:

This is a well explained request: Why don't you wrap all these lines of code inside of an <header> element, that would improve the HTML structure and make the page more accessible

This request is not well explained: You would like to wrap all these lines of code inside of an <header> element for semantic reasons.

**Review by julie-ify — APPROVED** [review]
## Hi @helio3197, 

# Highlights
- Good job implementing the required changes from the previous review ✔️ 
- Please next time try creating a new review instead of editing the ones you made before okay so that reviewers could see your previous reviews 😄.

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved emoji

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**